src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersion.cs

```diff
@@ -58,5 +58,6 @@ public enum SchemaVersion
         V46 = 46,
         V47 = 47,
         V48 = 48,
+        V49 = 49,
```

Comment:
Heads up: I believe, due the issue with export and the changes Jared is actively working, version 49 will be released this week. Probably we should include all these changes under version 50.

Reply:
Sure. There is nothing to discuss as Jared's fix will go first.